bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -720,10 +720,10 @@ object TimeProblems {
 }
 
 object ResourceIssues {
-  def resourceNotOfferedCheck[B <: BlueprintBase](p: Proposal, s: ShellAdvisor, sem: Semester, name: Option[String] = None): List[Problem] = for {
+  def resourceNotOfferedCheck[B <: BlueprintBase : Manifest](p: Proposal, s: ShellAdvisor, sem: Semester, name: Option[String] = None): List[Problem] = for {
```

Comment:
Isn't `Manifest` kind of deprecated, preferring `TypeTag` instead?

Reply:
Hm, `b.isInstanceOf[B]` should have been causing a loud warning.

```
scala> def foo[A](x: Any): Boolean = x.isInstanceOf[A] 
<console>:11: warning: abstract type A is unchecked since it is eliminated by erasure
       def foo[A](x: Any): Boolean = x.isInstanceOf[A]
                                                   ^
foo: [A](x: Any)Boolean

scala> 
```

This is all terrible but :+1:. If this codebase had a future I would suggest we figure out a way to do this more safely.

@cquiroz in this case I think a class check is probably fine. With a `TypeTag` you could use a type pattern and do an exact match but you really only need that if `B` is a parameterized type.